readme.md

```diff
@@ -124,7 +145,12 @@ plugin is able to draw Swagger-template to display the generated documentation f
  - *auto-doc.route* - it's a route for generated documentation  
  - *auto-doc.basePath* - it's a root of your api root
  
-Also you can specify way to collect documentation by creating your custom data collector class.
+Also, you can specify way to collect documentation by creating your custom data collector class.
+
+### Test result
+
+![img.png](assets/images/img.png)
```

Comment:
add the steps before:

4. Run tests
5. Go to route defined in config `auto-doc.route`
6. Profit!

And paster the image here.